recipes/xgcm/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: xgcm
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/x/xgcm/xgcm-{{ version }}.tar.gz
+  sha256: 1eb5e716530b63fbe7da129fd20fa85451c527f61230df172e419eb100ce2f63
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+    - pytest-runner
+  run:
+    - python
+    - xarray
+    - dask
+    - numpy
+    - future
+    - docrep
+
+test:
+  imports:
+    - xgcm
+
+about:
+  home: https://github.com/xgcm/xgcm
+  license: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

Comment:
The tarball does not package the license file. We need a `MANIFEST.in` PR to fix that.

Reply:
Raised issue ( https://github.com/xgcm/xgcm/issues/101 ) on this point.